tests/cephadm/test_validate_osd_pool.py

```diff
@@ -0,0 +1,39 @@
+from ceph.ceph_admin.common import fetch_method
+from ceph.ceph_admin.helper import get_cluster_state
+from ceph.ceph_admin.osd import OSD
+from utility.log import Log
+
+log = Log(__name__)
+
+
+CLUSTER_STATE = [
+    "ceph osd df",
+    "ceph osd tree",
+]
+
+
+def run(ceph_cluster, **kw):
+    """
+    Ceph-admin module to manage ceph-osd service
+
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): Ceph cluster object
+        kw: test data
+
+    check ceph.ceph_admin.osd for test config
+    """
+    config = kw.get("config")
+
+    # Manage Ceph using ceph-admin orchestration
+    command = config.pop("command")
+    log.info("Executing service %s service" % command)
```

Comment:
The command passed is `ceph osd pool ls detail`... which is not a service. Could we fix the log message.